apps/frontend/src/App.test.tsx

```diff
@@ -0,0 +1,5 @@
+import { expect, test } from "vitest";
```

Comment:
We didn't really plan on using vitest, but it apperantly works fine with cypress installed so this is likely fine, however in ci we'll need to configure it to run our cypress tests

Reply:
`vitest` is the `vite` native way of writing unit tests. It looks like you can use Cypress for unit tests, so if you want we can switch to just using Cypress (instead of both). If wanted, that'll be an easy swap, so I'm going to go ahead and merge this PR and we can open a follow up ticket if required.